src/jstests/formio/components/time.spec.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import _ from 'lodash';
+import {Formio} from 'react-formio';
+import {timeForm} from './fixtures/time';
+import OpenFormsModule from '../../../formio/module';
+
+jest.mock('../../../map/rd', () => jest.fn());
```

Comment:
I think this was no longer needed :)

Reply:
Aaah yes, bad copy and paste on my part! I will remove it also from the IBAN tests so I don't accidentally copy it anymore :D